### PR TITLE
update new bike creation and fix layouts for table

### PR DESCRIPTION
**Pull request by mikemitchel**
This replaces the hard to see purple message with the new message toast interface. It also clears the new bike form on success to make it easier to add several bikes in a row

### CODE REVIEW

File: src/lib/ui/utils/formatters.ts [hunk 1]
```diff
@@ -6,7 +6,16 @@ export const getDisplayName = (member?: Member): string => {
...
+	const options: Intl.DateTimeFormatOptions = {
+		year: 'numeric',
+		month: 'numeric',
+		day: 'numeric',
+		hour: '2-digit',
+		minute: '2-digit',
+		hour12: true
+	};
+
+	return date ? date.toLocaleString(undefined, options) : ''; // undefined uses browswer default locale
```
Comment (BigAB) [inline src/lib/ui/utils/formatters.ts:18]: I am not sure this works, you probably have to do 
```ts
new Intl.DateTimeFormat(['en-us', 'en-ca'], {
  year: 'numeric',
  month: 'numeric',
  day: 'numeric',
  hour: '2-digit',
  minute: '2-digit',
  hour12: true
}).format(date)
```

**Review by BigAB — COMMENTED** [review]
This can look a little odd at smaller screen sizes:

localhost_5173_bikes

**Comment by mikemitchel** [comment]
before:

ipadb4

after:


ipad after

File: src/lib/ui/utils/formatters.ts (hunk 1 above)
Comment (mikemitchel) [reply src/lib/ui/utils/formatters.ts]: i changed the way this works to read from the config file

**Review by BigAB — COMMENTED** [review]
The new changes don't seem to address the original idea of removing the horizontal scroll bar on the table when possible.

Can we either have the table break out of the `<main>` column, and only have scrollbars when it exceeds the "screen width - 1 rem" padding or something?

**Review by BigAB — APPROVED** [review]

**Pull request merged**